### PR TITLE
Version 2.1 release

### PR DESCRIPTION
**Pull request by stefpiatek**
As with the other repos in this release, didn't increment the versions on develop and then new changes have been merged into develop since starting the most recent live run. Created branch from versions running on live, and will merge into master, then will merge master into develop to correct for the right version and avoid merge conflicts.

### CODE REVIEW

File: inform-annotations/pom.xml [hunk 1]
```diff
@@ -3,15 +3,15 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<artifactId>inform-annotations</artifactId>
-	<version>1.0.0</version>
+	<version>2.0</version>
```
Comment (klapaukh) [inline inform-annotations/pom.xml:6]: Should be 2.1?

File: inform-annotations/pom.xml [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 	<name>Inform Annotations</name>
 	<packaging>jar</packaging>
 	<description>Inform-DB Spring JPA Annotation Helpers</description>
 
 	<parent>
 		<groupId>uk.ac.ucl.rits.inform</groupId>
 		<artifactId>emap-star</artifactId>
-		<version>1.0</version>
+		<version>2.0</version>
```
Comment (klapaukh) [inline inform-annotations/pom.xml:14]: Should this also be 2.1?

File: inform-db/pom.xml [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
+	<version>2.1</version>
 	<name>Inform-DB</name>
 	<packaging>jar</packaging>
 	<description>Inform-DB Spring JPA classes</description>
 
 	<parent>
 		<groupId>uk.ac.ucl.rits.inform</groupId>
 		<artifactId>emap-star</artifactId>
-		<version>1.0</version>
+		<version>2.0</version>
```
Comment (klapaukh) [inline inform-db/pom.xml:14]: Don't they move in sync?

File: pom.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
 	<groupId>uk.ac.ucl.rits.inform</groupId>
 	<artifactId>emap-star</artifactId>
 	<packaging>pom</packaging>
-	<version>1.0</version>
+	<version>2.0</version>
```
Comment (klapaukh) [inline pom.xml:9]: I guess if the children change the parent should also go to 2.1?

**Review by klapaukh — APPROVED** [review]

File: inform-annotations/pom.xml (hunk 2 above)
Comment (stefpiatek) [reply inform-annotations/pom.xml:14]: ah interesting, for this one I thought the annotations hadn't changed but yeah makes it clearer for everything to move in step

**Pull request merged**